plane/source/docs/automatic-landing.rst

```diff
@@ -172,26 +134,26 @@ small negative number can be good, to allow the nose to be kept down a
 small amount to reduce the chance of stall if the flare happens too far
 off the ground.
 
-Note that the actual pitch of the aircraft can be quite a bit above
-``LAND_PITCH_CD`` as the TECS controller tries to control the descent
+Note that the actual pitch of the aircraft can be quite a bit above :ref:`LAND_PITCH_CD <LAND_PITCH_CD>` as the TECS controller tries to control the descent
 rate. The maximum pitch is controlled by the
 :ref:`TECS_PITCH_MAX <TECS_PITCH_MAX>`
 parameter if it is non-zero, otherwise by the
 :ref:`LIM_PITCH_MAX <LIM_PITCH_MAX>` parameter.
 
-The ``TECS_LAND_DAMP`` parameter is a damping constant for the pitch
+The :ref:`TECS_LAND_DAMP<TECS_LAND_DAMP>` parameter is a damping constant for the pitch
```

Comment:
It would be useful to have some recommendations for good values for this. The default seems to be 0.5. I'm setting up an X-Fly Glastar with no airspeed sensor. Is the recommendation to not change this unless there is a problem?

Reply:
yep,dont change...there is a new param in master that tweaks flare touchdown aim point better...its what actually got me to start work on this...to get the stable stuff understandable... then will add the new stuff in upcoming